src/app.js

```diff
@@ -33,6 +34,7 @@ if(config.MODE == 'development') {
 
 app.use('/', indexRoute);
 app.use('/users', userRoute);
+app.use('/register', authRoute);
```

Comment:
```suggestion
app.use('/', authRoute);
```

ini sebaiknya `register` nya ditaruh di dalam, soalnya nanti authRoute akan punya api `/login` & `/logout` juga kan?